docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -553,32 +566,34 @@ public class CodeFlowIT extends CodeFlowTest {
 }
 ----
 
-Please see xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for more information about the way it is initialized and configured.
+For more information about how it is initialized and configured, see xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak].
 
-=== Use Browser
+=== Use the browser
 
-To test the application, you should open your browser and access the following URL:
+To test the application, open your browser and access the following URL:
 
 * http://localhost:8080/default[http://localhost:8080/default]
 
-If everything is working as expected, you should be redirected to the Keycloak server to authenticate. Note that the requested path
-defines a `default` tenant which we don't have mapped in the configuration file. In this case, the default configuration will be used.
+If everything works as expected, you are redirected to the Keycloak server to authenticate.
+Be aware that the requested path defines a `default` tenant, which we don't have mapped in the configuration file.
+In this case, the default configuration is used.
 
-In order to authenticate to the application you should type the following credentials when at the Keycloak login page:
+To authenticate to the application, enter the following credentials in the Keycloak login page:
 
-* Username: *alice*
-* Password: *alice*
+* Username: `alice`
+* Password: `alice`
 
-After clicking the `Login` button you should be redirected back to the application.
+After clicking the *Login* button, you are redirected back to the application.
 
 If you try now to access the application at the following URL:
 
 * http://localhost:8080/tenant-a[http://localhost:8080/tenant-a]
 
-You should be redirected again to the login page at Keycloak. However, now you are going to authenticate using a different `realm`.
+You are redirected again to the Keycloak login page.
+However, now you are going to authenticate by using a different `realm`.
```

Comment:
```suggestion
This time, the redirection to the Keycloak login page requires authentication using a different `realm`.
```

Reply:
IMHO it  is not quite correct, `the redirection` does not require it, it is only a tool the browser uses to bring the user to the login screen

Reply:
Thank you. Partially updated.